mollie/api/resources/methods.py

```diff
@@ -66,7 +65,7 @@ def enable(self, method_id: str, **params: Any) -> Method:
         resource_path = self.get_resource_path()
         path = f"{resource_path}/{method_id}"
         result = self.perform_api_call(self.REST_CREATE, path, params=params)
-        return self.get_resource_object(result)
+        return Method(result, self.client)
```

Comment:
In these instances I chose to be explicit in the class instantiation, alternatively `Method` could be replaced by `self.object_type`